### PR TITLE
feat(website): setup eslint, tailwind and proper temp homepage

### PR DESCRIPTION
**Pull request by ajmnz**
(PNG Image, 1440 × 790 pixels)

### CODE REVIEW

**Review by TheOtterlord — APPROVED** [review]
looks awesome :)

File: website/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules/
+
+*.spec.tsx
+*.spec.tsx.snap
+src/generated/
```
Comment (amitojsingh366) [inline website/.eslintignore:5]: idk what this does so im not gonna merge, ill leave that to others

**Review by amitojsingh366 — APPROVED** [review]
looks good to me!

**Comment by HoloPanio** [comment]
Preview: https://duxcore-esu5g0yus-holopanio.vercel.app/

**Review by HoloPanio — APPROVED** [review]
Looks good to me.  :+1:

**Review by overlisted — APPROVED** [review]
lgtm

**Comment by HoloPanio** [comment]
testing something out, please ignore this comment

**Pull request merged**